tools/erc-repository-indexer/erc-contract-indexer/src/services/registryGenerator.ts

```diff
@@ -102,38 +136,38 @@ export class RegistryGenerator {
     // Convert Map values back to array for file writing
     const uniqueContracts = Array.from(contractMap.values());
 
-    await this.writeContractsToFile(filePath, uniqueContracts);
+    await this.writeContentsToFile(filePath, uniqueContracts);
+    console.log('Finish pushing new ERC token contracts to registry.');
```

Comment:
nit: we should say how many were written
```suggestion
    console.log('Finished writing x new ERC token contracts to registry.');
```

Reply:
Gotcha yeah good point! Will update in follow up PRs